dbt/tests/adapter/caching/test_caching.py

```diff
@@ -116,3 +116,7 @@ class TestCachingUppercaseModel(BaseCachingUppercaseModel):
 
 class TestCachingSelectedSchemaOnly(BaseCachingSelectedSchemaOnly):
     pass
+
+
+class TestNoPopulateCache(BaseNoPopulateCache):
```

Comment:
thinking out loud: do we actually need any of the `Test`-prefixed integration tests in dbt-adapters? as in, will they actually be run against a concrete adapter as part of dbt-adapters development? If not, I wonder if we should just remove them and only define `Base` integration tests in this repo.

Reply:
I agree that we don't need them. In cases where they already exist, I kept them in case folks were importing them as-is (causing them to run as-is). But we don't need to add new ones. I did that because that was the pattern that we had. I can remove the net new ones if that's preferable.